tests/e2e/workloads/bdi/test_bdi_workload.py

```diff
@@ -0,0 +1,249 @@
+        )
+
+        log.info("Creating Job for for configure-workload")
+        ocp_jobs.create(yaml_file=self.temp_yaml_configure.name)
+        ocp_jobs.wait_for_resource(
+            condition="1/1",
+            resource_name="db2u-configure-workload-job",
+            column="COMPLETIONS",
+            timeout=self.configure_timeout,
+            sleep=30,
+        )
+        now = datetime.now()
+
+        current_time = now.strftime("%H:%M:%S")
+        log.info(
+            f"##################### Finished configure-workload at: {current_time} #######################"
+        )
+
+        # DATA WORKLOAD
+        self.temp_data_load_dict = init_data_load_yaml(
+            namespace=self.db2u_project,
+            image=self.db2u_image_url,
+            pvc_name=self.db2u_pvc_name,
+            secret_name=self.db2u_secret_name,
+        )
+        self.temp_yaml_data = tempfile.NamedTemporaryFile(
+            mode="w", prefix="data_yaml_", delete=False
+        )
+        templating.dump_data_to_temp_yaml(
+            self.temp_data_load_dict, self.temp_yaml_data.name
+        )
+
+        log.info("Creating Job for for data-load")
+        ocp_jobs.create(yaml_file=self.temp_yaml_data.name)
+        ocp_jobs.wait_for_resource(
+            condition="1/1",
+            resource_name="db2u-data-load-job",
+            column="COMPLETIONS",
+            timeout=self.data_load_timeout,
+            sleep=30,
+        )
+
+        now = datetime.now()
+
+        current_time = now.strftime("%H:%M:%S")
+        log.info(
+            f"##################### Finished data-load at: {current_time} #######################"
+        )
+
+        # RUN WORKLOAD
+        self.temp_run_dict = init_run_workload_yaml(
+            namespace=self.db2u_project,
+            image=self.db2u_image_url,
+            pvc_name=self.db2u_pvc_name,
+            secret_name=self.db2u_secret_name,
+        )
+        self.temp_yaml_run = tempfile.NamedTemporaryFile(
+            mode="w", prefix="run_yaml_", delete=False
+        )
+        templating.dump_data_to_temp_yaml(self.temp_run_dict, self.temp_yaml_run.name)
+
+        log.info("Creating Job for for run-workload")
+        ocp_jobs.create(yaml_file=self.temp_yaml_run.name)
+        ocp_jobs.wait_for_resource(
+            condition="1/1",
+            resource_name="db2u-run-workload-job",
+            column="COMPLETIONS",
+            timeout=self.run_workload_timeout,
+            sleep=30,
+        )
+        now = datetime.now()
+
+        current_time = now.strftime("%H:%M:%S")
+        log.info(
+            f"##################### Finished run-workload at: {current_time} #######################"
+        )
+
+
+class TestBdiWorkloadSF1(TestBdiWorkloadBaseClass):
+    """
+    Tests BDI workload on SF (Scale Factor) = 1
+    """
+
+    def test_bdi_workload(self):
+        self.run()
+
+
+class TestBdiWorkloadSF10(TestBdiWorkloadBaseClass):
```

Comment:
Can we use parametrize to run below three tests? wdyt?

Reply:
As I wrote in one of the previous comments, we need to re-assess the usage of this tool and change it accordingly.